src/cloud-cli/meltano/cloud/api/auth/auth.py

```diff
@@ -3,18 +3,20 @@
 from __future__ import annotations
 
 import asyncio
-import os
-import platform
-import subprocess
 import sys
+import tempfile
 import typing as t
 import webbrowser
-from contextlib import contextmanager
+from contextlib import asynccontextmanager
+from http import HTTPStatus
 from pathlib import Path
 from urllib.parse import urlencode, urljoin
 
 import aiohttp
 import click
+import importlib_resources
```

Comment:
Wdyt about falling back to `importlib.resources` for Python 3.9+:

```python

if sys.version_info < (3, 9):
    import importlib_resources
else:
    import importlib.resources as importlib_resources
```

That way, it'll eventually be auto-removed by Ruff/pyupgrade.

Reply:
Implemented in 9ac790e. Had to make a change from what you suggested to satisfy WPS301.